src/core/datastructures/volume/volume.cpp

```diff
@@ -178,13 +178,13 @@ Document Volume::getInfo() const {
 }
 
 vec3 Volume::getWorldSpaceGradientSpacing() const {
-    mat3 textureToWorld = mat3(getCoordinateTransformer().getTextureToWorldMatrix());
+    const auto textureToWorld = mat3(getCoordinateTransformer().getTextureToWorldMatrix());
     // Basis vectors with a length of one voxel.
     // Basis vectors may be non-orthogonal
-    auto dimensions = getDimensions();
-    vec3 a = textureToWorld[0] / static_cast<float>(dimensions[0]);
-    vec3 b = textureToWorld[1] / static_cast<float>(dimensions[1]);
-    vec3 c = textureToWorld[2] / static_cast<float>(dimensions[2]);
+    const auto dimensions = getDimensions();
```

Comment:
Is this really necessary? Seems very verbose and the compiler can easily deduce it anyway. Will need to change almost entire codebase in that case?

Reply:
You mean the const? Yes we want that everywhere.